lib/components/DrawerFooter/index.tsx

```diff
@@ -1,51 +1,53 @@
 export const DrawerFooter = () => {
   return (
     <div class="artajs__tracking__footer__wrapper">
-      <div class="artajs__tracking__footer__content">
-        <div class="artajs__tracking__footer__powered__by__text">
-          Powered by
-        </div>
-        <div class="artajs__tracking__footer__powered__by__logo">
+      <a href="https://arta.io/">
```

Comment:
Updating the "Powered by" sections in both Estimates and Tracking to be more consistent